### PR TITLE
Configurable regime precedence

### PR DESCRIPTION
**Pull request by eligrey**
## Related Issues

- https://transcend.height.app/T-18952
- Closes https://transcend.height.app/T-19214
- Closes https://transcend.height.app/T-19217

## Security Implications

_[none]_

## System Availability

_[none]_

### CODE REVIEW

File: src/regimes.ts [hunk 1]
```diff
@@ -1,17 +1,13 @@
-import {
-  PrivacyRegime,
-  PrivacyRegimeEnum,
-} from '@transcend-io/airgap.js-types';
+import { PrivacyRegime } from '@transcend-io/airgap.js-types';
+import { settings } from './settings';
+import { COMMA_AND_OR_SPACE_SEPARATED_LIST } from './utils/comma-and-or-space-separated-list';
+
+const { regimePrecedence = 'GDPR LGPD CPRA CDPA CPA Unknown' } = settings;
```
Comment (anotherminh) [inline src/regimes.ts:5]: Do we not need to add `regimePrecedence` to be a new setting in the settings file?

**Review by anotherminh — APPROVED** [review]

File: src/regimes.ts (hunk 1 above)
Comment (eligrey) [reply src/regimes.ts:5]: This only needs to be added to ag-types in the monorepo for backend integration.

Comment (anotherminh) [reply src/regimes.ts:5]: Can you add a Height task for that/make sure to add it as well?

Comment (michaelfarrell76) [reply src/regimes.ts:5]: ya this should definitely be something backend-specified

**Pull request merged**